src/components/article/Introduction.js

```diff
@@ -8,10 +8,18 @@ import React, { Component } from 'react'
 // lodash
 import get from 'lodash/get'
 
-export const Introduction = ({ data }) => {
+export const Introduction = ({ data, fontSize }) => {
   let content
   let block = false
   let introArr = []
+  let theStyle
+  if(fontSize ==='large') {
+    theStyle = styles['font-enlarge']
+  }else if(fontSize ==='medium') {
+    theStyle = styles['font-default']
+  }else {
+    theStyle = styles['font-shrink']
```

Comment:
Read [classnames](https://github.com/JedWatson/classnames) doc, and rewrite these lines.